tensorflow/src/main/java/org/bytedeco/javacpp/presets/tensorflow.java

```diff
@@ -41,40 +35,91 @@
+        "tensorflow/core/platform/init_main.h",
+        "tensorflow/core/platform/types.h",
+        "tensorflow/core/platform/mutex.h",
+        "tensorflow/core/platform/macros.h",
+        "tensorflow/core/util/port.h",
+        "tensorflow/core/lib/core/error_codes.pb.h",
+        "tensorflow/core/platform/logging.h",
+        "tensorflow/core/lib/core/status.h",
+        "tensorflow/core/platform/protobuf.h",
+        "tensorflow/core/platform/file_system.h",
+        "tensorflow/core/platform/file_statistics.h",
+        "tensorflow/core/platform/env.h",
+        "tensorflow/core/graph/dot.h",
+        "tensorflow/core/protobuf/config.pb.h",
+        "tensorflow/core/framework/cost_graph.pb.h",
+        "tensorflow/core/framework/step_stats.pb.h",
+        "tensorflow/core/framework/versions.pb.h",
+        "tensorflow/core/public/session_options.h",
+        "tensorflow/core/lib/core/threadpool.h",
+        "tensorflow/core/framework/allocation_description.pb.h",
+        "tensorflow/core/framework/allocator.h",
+        "tensorflow/core/framework/tensor_shape.pb.h",
+        "tensorflow/core/framework/types.pb.h",
+        "tensorflow/core/framework/tensor.pb.h",
+        "tensorflow/core/framework/tensor_description.pb.h",
+        "tensorflow/core/framework/tensor_types.h",
+        "tensorflow/core/framework/tensor_shape.h",
+//        "tensorflow/core/framework/tensor_slice.h",
+        "tensorflow/core/framework/tensor_util.h",
+        "tensorflow/core/framework/tensor_reference.h",
+        "tensorflow/core/framework/tensor.h",
+        "tensorflow/core/framework/attr_value.pb.h",
+        "tensorflow/core/framework/op_def.pb.h",
+        "tensorflow/core/framework/function.pb.h",
+        "tensorflow/core/framework/graph.pb.h",
+        "tensorflow/core/framework/shape_inference.h",
+        "tensorflow/core/framework/partial_tensor_shape.h",
+        "tensorflow/core/public/session.h",
+        "tensorflow/c/c_api.h",
+        "tensorflow/core/framework/op_def.pb.h",
+        "tensorflow/core/framework/op_def_builder.h",
+        "tensorflow/core/framework/op_def_util.h",
+        "tensorflow/core/framework/op.h",
+        "tensorflow/core/framework/types.h",
+        "tensorflow/core/graph/edgeset.h",
+        "tensorflow/core/lib/gtl/iterator_range.h",
+//        "tensorflow/core/lib/gtl/inlined_vector.h",
+        "tensorflow/core/graph/graph.h",
+        "tensorflow/core/framework/node_def_builder.h",
+        "tensorflow/core/framework/node_def_util.h",
+        "tensorflow/core/framework/selective_registration.h",
+        "tensorflow/core/graph/node_builder.h",
+        "tensorflow/core/graph/graph_def_builder.h",
+        "tensorflow/core/graph/default_device.h",
+        "tensorflow/core/graph/graph_constructor.h",
+        "tensorflow/cc/framework/scope.h",
+        "tensorflow/cc/framework/ops.h",
+        "tensorflow/cc/framework/cc_op_gen.h",
+        "tensorflow_adapters.h"},
+        link = "tensorflow"),
```

Comment:
@mirosval: I think I found it. The build worked when I changed to link agains `tensorflow_cc`. I guess this is because you had to switch to build libtensorflow_cc instead of libtensorflow which it was in version 0.9.